[2023-03-12T22:42:31.026+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: validacao_pu.task_calcular_pu_externo_dfg scheduled__2023-03-02T00:00:00+00:00 [queued]>
[2023-03-12T22:42:31.092+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: validacao_pu.task_calcular_pu_externo_dfg scheduled__2023-03-02T00:00:00+00:00 [queued]>
[2023-03-12T22:42:31.093+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-12T22:42:31.094+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2023-03-12T22:42:31.095+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-12T22:42:31.262+0000] {taskinstance.py:1300} INFO - Executing <Task(_PythonDecoratedOperator): task_calcular_pu_externo_dfg> on 2023-03-02 00:00:00+00:00
[2023-03-12T22:42:31.274+0000] {standard_task_runner.py:55} INFO - Started process 1460 to run task
[2023-03-12T22:42:31.305+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'validacao_pu', 'task_calcular_pu_externo_dfg', 'scheduled__2023-03-02T00:00:00+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/examples/validacao_pu.py', '--cfg-path', '/tmp/tmptkrxdrmb']
[2023-03-12T22:42:31.305+0000] {standard_task_runner.py:83} INFO - Job 94: Subtask task_calcular_pu_externo_dfg
[2023-03-12T22:42:31.547+0000] {task_command.py:388} INFO - Running <TaskInstance: validacao_pu.task_calcular_pu_externo_dfg scheduled__2023-03-02T00:00:00+00:00 [running]> on host 40565fc59ef6
[2023-03-12T22:42:31.879+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alexandre.yoshimatsu
AIRFLOW_CTX_DAG_ID=validacao_pu
AIRFLOW_CTX_TASK_ID=task_calcular_pu_externo_dfg
AIRFLOW_CTX_EXECUTION_DATE=2023-03-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-02T00:00:00+00:00
[2023-03-12T22:42:31.880+0000] {logging_mixin.py:137} INFO - Operação: dfg
[2023-03-12T22:42:31.880+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-03-12T22:42:31.908+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=validacao_pu, task_id=task_calcular_pu_externo_dfg, execution_date=20230302T000000, start_date=20230312T224231, end_date=20230312T224231
[2023-03-12T22:42:32.002+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-12T22:42:32.087+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
